### PR TITLE
Improve proposals in deposit period

### LINKED ISSUE #3079: deposit improvements
https://app.lunie.io/proposals/18

- `Voting Start Date - December 31st 0000`, 18:42 is not correct
- this page should show when the deposit period is over

- ~"address you are trying to send to is not in the extension"~
- ~not spaced properly in the modal - too close to the steps~
- ~the error message should include an action (add it to the extension or choose another signing option)~
- ~this is not a send tx~

Screen Shot 2019-11-01 at 12 45 09 PM

### PR DESCRIPTION
**Pull request by mariopino**
Closes #3079 

**Description:**

- Display Deposit Period End Date correctly for proposals in deposit period (related: https://github.com/luniehq/lunie-api/pull/181)
- Fix no proposal status badge showed when refreshing the proposal page.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/governance/LiProposal.vue [hunk 1]
```diff
@@ -94,35 +94,4 @@ export default {
...
-  color: var(--success);
-  border-color: var(--success);
-}
-
-.proposal-status.votingperiod {
-  color: var(--tertiary);
-  border-color: var(--tertiary);
-}
-</style>
+<style scoped src="../../styles/proposal.css"></style>
```
Comment (jbibla) [inline src/components/governance/LiProposal.vue]: i would propose importing the css instead `@import "../../styles/proposal.css";`

File: src/styles/proposal.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.proposal-status {
```
Comment (jbibla) [inline src/styles/proposal.css]: i might consider renaming this file `proposal-status.css` but no big deal

**Comment by jbibla** [comment]
> Display Deposit Period End Date correctly for proposals in deposit period

is this in this PR?

> Fix no proposal status badge showed when refreshing the proposal page.

might be time for a status component - but this is a good fix for now!

**Comment by mariopino** [comment]
> is this in this PR?

Yep
 
> might be time for a status component - but this is a good fix for now!

Agree!

**Comment by faboweb** [comment]
@mariopino let's fix the commented things and merge this

File: src/components/governance/LiProposal.vue (hunk 1 above)
Comment (mariopino) [reply src/components/governance/LiProposal.vue]: Yep `<style scoped src="../../styles/proposal.css"></style>` works, I was testing that.

**Comment by faboweb** [comment]
awesome! nice refactor and small and simple PR

**Pull request merged**